### DISCUSSION TITLE
How to change SEO attributes (lang, dir html attribute) with no prefix strategy

### DISCUSSION THREAD

**Discussion by mini-op** [nuxt-modules/i18n#2840] [Q&A]
I have this in my app.vue for direction and language.
![image](https://github.com/nuxt-modules/i18n/assets/77070946/8e754891-64d2-4b02-a82a-d001b0104d61)

while using strategy **prefix** and changing the locale with `setLocale()` it will be updated 
as expected.

however, with `no prefix` strategy it seems it's not reactive or its not getting updated.

is this how it supposed to be? If yes, what's the best way to add a reactive lang and a dir attribute to html?

**Comment by iqra123ali** [comment]
SEO (Search Engine Optimization) is the practice of optimizing a website to increase its visibility and ranking on search engine results pages, ultimately driving organic (non-paid) traffic to the site.

Ideoversity Instituhttps://ideoversity.com/te is an educational institution dedicated to teaching innovative techniques and strategies in digital marketing, with a focus on SEO, helping individuals and businesses stay ahead in the ever-evolving online landscape.
